R/tokens_xptr.R

```diff
@@ -83,7 +83,28 @@ types.tokens_xptr <- function(x) {
 
 #' @export
 ntoken.tokens_xptr <- function(x, ...) {
-    structure(cpp_ntoken(x), names = docnames(x))
+    if (length(list(...))) {
```

Comment:
It should be like
```
structure(cpp_ntoken(tokens(x, ...)), names = docnames(x))
```

Reply:
That was the first thing that I tried, but it then fails the test if called a second time, since a call with any `...` options modifies x outside of the function scope. If this function body is just replaced with that code, it will fail in `test_that("n functions works the same on both tokens and tokens_xptr"` - see my note there.